### ISSUE TITLE
breadcrumb error in about pages

### ISSUE THREAD

**Issue by jacobthill** [sul-dlss/dlme#1267]
If I select 'Edit' in the first screenshot and then 'About pages' in the second screenshot, the breadcrumb in the third screenshot adds 'Curation'. It also introduces the 'Curator' heading on top of the 'About' heading (screenshots two and three) and adds the curator side menu (screenshot three).
<img width="1133" alt="Screen Shot 2021-05-19 at 9 15 16 AM" src="https://user-images.githubusercontent.com/37662787/118819061-f7590e00-b882-11eb-82fb-ceb058623b92.png">
<img width="1051" alt="Screen Shot 2021-05-19 at 9 15 28 AM" src="https://user-images.githubusercontent.com/37662787/118819908-caf1c180-b883-11eb-8ecc-19b518e4daa2.png">
<img width="1099" alt="Screen Shot 2021-05-19 at 9 15 35 AM" src="https://user-images.githubusercontent.com/37662787/118819920-cc22ee80-b883-11eb-84d9-00abd5ec4aae.png">

**Comment by ggeisler** [comment]
@jacobthill To be clear, the bug you perceive here is that in the second screenshot, when you select the "About pages" link in the breadcrumb, you expect to be returned to the visitor view of the About page (similar to if you had selected "Cancel" at the bottom of the edit page form)?

I think the ideal fix here is to make sure we add the "Curation" part of the breadcrumb path when we're editing a page (would also apply to feature pages, though there we also have the extra complication of an additional breadcrumb node because of the two-level option of feature pages), since you are in Curation mode at that point.

In other words, it isn't that the link is directing to the wrong place or that the "Curation" heading is inappropriate, it's that the breadcrumb path is not fully updated when in the state of editing a page.

Not sure how complicated this will be, and it only affects curators in kind of an edge case scenario (curator choosing to use the breadcrumb to get out of edit mode rather than using the Cancel or Save buttons), but I believe it is a general Spotlight issue.

**Comment by jacobthill** [comment]
I'm not sure what I should expect to be honest. I just noticed that when I selected 'About pages' it added 'Curation' to the breadcrumbs. From my point of view this is not an issue. I can do what I need to do as is. I just wanted to surface this for you all to be aware of. Feel free to move the issue to the Spotlight repo and/or close it as you see fit.

**Comment by ggeisler** [comment]
Okay, sounds good, thanks. I'll write up a Spotlight ticket to capture the issue and close this one when I do.

**Comment by ggeisler** [comment]
@jacobthill After starting to write this up in Spotlight, I'm inclined to close it and not worry about it unless someone else reports it in the future.

You have a totally valid concern, but inserting a new node for "Curation" in the breadcrumb as I suggested above, while theoretically the right thing to do, is tricky because the associated link (e.g., "https://exhibits.stanford.edu/exhibit-title/dashboard/curation") does not currently exist; we don't have a page to resolve to for a link that ends in "/curation". 

So the fix would be a little more involved and I really think since this is an edge case, that only affects curators, it wouldn't be a very high priority to fix and I'm not sure is worth the time right time to figure out how to solve it.
